about.html

```diff
@@ -0,0 +1,84 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>End of the World Event</title>
+    <link rel="icon" type="image/x-icon" href="ressources/event-image/EoW-red-circle.png">
+
+  </head>
+  <body>
+    <header class="header">
+      <section class="black-banner-container">
+        <div class="black-banner-items">
+          <a href="https://facebook.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-facebook"></a>
+          <a href="https://twitter.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-twitter"></a>
+          <a href="#English" target="_blank" rel="noopener">English</a>
+          <a href="#MyPage" target="_blank" rel="noopener">My Page</a>
+          <a href="#Logout" target="_blank" rel="noopener">Logout</a>
+        </div>
+      </section>
+      <section class="white-banner">
+        <div class="logo-container">
+          <a href="/" class="logo-container-2">
+            <img src="ressources/event-image/EoW-Logo.png" alt="A red logo for the End of the World Event" class="logo">
+          </a>
+        </div>
+        <div class="menu-container">
+          <div id="overlay-screen" class="overlay">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <nav class="overlay-content menu">
+              <a href="./about.html" class="active">About</a>
+              <a href="#Program">Program</a>
+              <a href="#Jobs">Jobs</a>
+              <a href="#Sponsor">Sponsor</a>
+              <a href="#News">News</a>
+            </nav>
+          </div>
+          <span class="hamburger" onclick="openNav()">&#9776;</span>
+        </div>
+        <nav class="nav-menu">
+          <a href="./about.html" class="active">About</a>
+          <a href="#Program">Program</a>
+          <a href="#Jobs">Jobs</a>
+          <a href="#Sponsor">Sponsor</a>
+          <a href="#News">News</a>
+        </nav>
+        <div class="button-container">
+          <button class="header-button" type="button" name="button">CC Campaing</button>
+        </div>
+      </section>
+    </header>
+    <section class="grey-background">
+      <div class="about-text">
+        <h2 class="page-title">End of the World Event 2022</h2>
+        <p class="about-intro">The end is near, and the world is doing one last event to finish lives of 7 billions people this year. for the occasion amazing VIP will accompagny you along the way to leave all together. Would you rather be alone and bored, or dance like there is no tomorrow and have no regrets ?</p>
+        <h5>Please don't try to resist the event, it's the last thing you will do in your life</h5>
+        <h6 href = "mailto: endtogether@endoftheworld.com">endtogether@endoftheworld.com</h6>
+      </div>
+    </section>
+    <section>
```

Comment:
- [ ] You have forgot to add the logo section of your page, please add it
![Screenshot from 2021-11-16 22-53-09](https://user-images.githubusercontent.com/62705947/142072037-f7b0f1d2-aa30-48c7-becb-74b6c0aabae2.png)

Reply:
Added as per design requirements

Reply:
Well done